core/src/main/java/com/scalar/db/config/DatabaseConfig.java

```diff
@@ -56,9 +57,11 @@ public class DatabaseConfig {
   public static final String CROSS_PARTITION_SCAN_FILTERING = SCAN_PREFIX + "filtering.enabled";
   public static final String CROSS_PARTITION_SCAN_ORDERING = SCAN_PREFIX + "ordering.enabled";
   public static final String SYSTEM_NAMESPACE_NAME = PREFIX + "system_namespace_name";
+  public static final String SCANNER_FETCH_SIZE = PREFIX + "scanner_fetch_size";
```

Comment:
Probably, it should be `scan_fetch_size` since a scanner is more like an iterator for results that are scanned with the specified fetch size?